src/assets/styles/_helper.scss

```diff
@@ -1,1726 +1,16 @@
-}
-
-.size-normal {
-  font-size: 100% !important;
-}
-
-.size-medium {
-  font-size: 125% !important;
-}
-
-.size-big,
-.size-large {
-  font-size: 150% !important;
-}
-
-.size-huge {
-  font-size: 200% !important;
-}
-
-.inherit {
-  font: inherit;
-}
-
-.no-wrap {
-  white-space: nowrap;
-}
-
-.auto-cell-size {
-  white-space: nowrap;
-  width: 1%;
-}
-
-.ls-0 {
-  letter-spacing: -3px;
-  margin-left: 10px;
-  margin-right: 10px;
-  white-space: nowrap;
-}
-
-.no-ul,
-.no-ul:hover,
-.no-ul a,
-.no-ul a:hover {
-  text-decoration: none;
-}
-
-.color-inherit {
-  color: inherit;
-}
-
-.clear {
-  clear: both;
-}
-
-.clear:after {
-  display: table;
-  content: " ";
-  clear: both;
-}
-
-.f-left {
-  float: left;
-}
-
-.f-right {
-  float: right;
-}
-
-.f-none {
-  float: none;
-}
-
-.block {
-  display: block !important;
-}
-
-.inline {
-  display: inline !important;
-}
-
-.in-block {
-  display: inline-block !important;
-  *display: inline !important;
-  zoom: 1;
-}
-
-.d-none,
-.hide,
-.hidden {
-  display: none !important;
-}
-
-.rel,
-.relative {
-  position: relative !important;
-}
-
-.abs,
-.absolute {
-  position: absolute;
-}
-
-.fixed {
-  position: fixed;
-}
-
-.t-0 {
-  top: 0;
-}
-
-.b-0 {
-  bottom: 0;
-}
-
-.l-0 {
-  left: 0;
-}
-
-.r-0 {
-  right: 0;
-}
-
-.tb-0 {
-  top: 0;
-  bottom: 0;
-}
-
-.lr-0 {
-  left: 0;
-  right: 0;
-}
-
-.v-mid,
-.v-m {
-  vertical-align: middle !important;
-}
-
-.v-top,
-.v-t {
-  vertical-align: top !important;
-}
-
-.v-bottom,
-.v-b {
-  vertical-align: bottom !important;
-}
-
-.v-super {
-  vertical-align: super !important;
-}
-
-.full-input input,
-.full-input select,
-.full-input textarea {
-  width: 100%;
-}
-
-.normal input,
-.normal select,
-.normal textarea,
-.full-input input[type="checkbox"],
-.full-input input[type="radio"] {
-  width: auto;
-}
-
-.no-shadow {
-  box-shadow: none !important;
-}
-
-.no-border-radius {
-  border-radius: 0 !important;
-}
-
-.overflow-x-scroll {
-  max-width: 100%;
-  overflow-x: auto;
-}
-
-.overflow-no {
-  overflow: hidden;
-}
-
-.overflow-auto {
-  overflow: auto;
-}
-
-.overflow-scroll {
-  overflow: scroll;
-}
-
-.relative {
-  position: relative;
-}
-
-.pointer {
-  cursor: pointer;
-}
-
-.fullwidth {
-  width: 100%;
-}
-
-.nolist {
-  list-style: none;
-}
-
-.text-shadow {
-  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.4);
-}
-
-.radius3 {
-  border-radius: 3px;
-}
-
-.radius5 {
-  border-radius: 5px;
-}
-
-.radius10 {
-  border-radius: 10px;
-}
-
-.text-rgb-2 {
-  color: rgba(0, 0, 0, 0.2);
-}
-
-.text-rgb-3 {
-  color: rgba(0, 0, 0, 0.3);
-}
-
-.text-rgb-4 {
-  color: rgba(0, 0, 0, 0.4);
-}
-
-.text-rgb-5 {
-  color: rgba(0, 0, 0, 0.5);
-}
-
-textarea {
-  resize: both;
-}
-
-textarea.vertical {
-  resize: vertical;
-}
-
-textarea.horizontal {
-  resize: vertical;
-}
-
-textarea.noresize {
-  resize: none;
-}
-
-.status-active {
-  color: $status-active;
-}
-
-.status-inactive {
-  color: $status-inactive;
-}
-
-.status-deleted {
```

Comment:
This should not be completely removed. Add it to another file like you did for the other classes.